### PR TITLE
Update Feedback/Env Var docs to follow style guide

### PR DESCRIPTION
**Pull request by marcysutton**
## Description

In looking at the new doc on [CLI Feedback](https://www.gatsbyjs.org/docs/cli-feedback/), I noticed it didn't follow the Gatsby Style Guide. I added a link to info about environment variables, and noticed that one had style guide issues as well. So this PR updates both of those docs to use the correct pronouns and improve some formatting.

### Documentation

N/A

## Related Issues

N/A

### CODE REVIEW

File: docs/docs/environment-variables.md [hunk 1]
```diff
@@ -67,8 +66,8 @@ In Windows it's a little more complex. [Check out this Stack Overflow article fo
 
 Project environment variables that you defined in the `.env.*` files will _NOT_ be immediately available
 in your Node.js scripts. To use those variables, use NPM package [dotenv](https://www.npmjs.com/package/dotenv) to
-examine the active `.env.*` file and attached those values,
-It's already a dependency of Gatsby, so you can require it in your `gatsby-config.js` or `gatsby-node.js` like this:
+examine the active `.env.*` file and attached those values.
```
Comment (laurieontech) [inline docs/docs/environment-variables.md]:
```suggestion
examine the active `.env.*` file and attach those values.
```

File: docs/docs/environment-variables.md [hunk 2]
```diff
@@ -67,8 +66,8 @@ In Windows it's a little more complex. [Check out this Stack Overflow article fo
...
+Dotenv is already a dependency of Gatsby, so you can require it in your `gatsby-config.js` or `gatsby-node.js` like this:
```
Comment (laurieontech) [inline docs/docs/environment-variables.md]:
```suggestion
`dotenv` is already a dependency of Gatsby, so you can require it in your `gatsby-config.js` or `gatsby-node.js` like this:
```

File: docs/docs/environment-variables.md [hunk 3]
```diff
@@ -136,26 +136,27 @@ Gatsby also allows you to specify another environment variable when running the
...
-Specifically, you can use `dotenv` and your individual OS Env Var to locate the `.env.myCustomEnvironment` file, and then use module.exports to store those Project Env Vars somewhere that the client-side JavaScript can access the values (via GraphQL queries).
 
-For instance: if you would like to add a `staging` environment with a custom Google Analytics Tracking ID, and a dedicated `apiUrl`. You can add `.env.staging` at the root of your project with the following modification to your `gatsby-config.js`
+Specifically, you can use `dotenv` and your individual OS Env Var to locate the `.env.myCustomEnvironment` file, and then use `module.exports` to store those Project Env Vars somewhere that the client-side JavaScript can access the values (via GraphQL queries).
+
+### Google Analytics env var example
 
-### Example
+If you would like to add a `staging` environment with a custom Google Analytics Tracking ID, and a dedicated `apiUrl`. You can add `.env.staging` at the root of your project with the following modification to your `gatsby-config.js`:
```
Comment (laurieontech) [inline docs/docs/environment-variables.md]:
```suggestion
If you would like to add a `staging` environment with a custom Google Analytics Tracking ID, and a dedicated `apiUrl` you can add `.env.staging` at the root of your project. In order to do so, use the following modification to your `gatsby-config.js`:
```

**Comment by marcysutton** [comment]
Thanks @laurieontech, I've added your suggestions!

**Comment by laurieontech** [comment]
Let's get this in!

**Pull request merged**